### PR TITLE
Prevent duplicate roles from being created

### PR DESCRIPTION
**Pull request by iFallUpHill**
Fixes https://github.com/JustYourPythonEnvironment/CheeseKimBot/issues/16

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-    "prefix": "kimbap ",
+    "prefix": "pimp ",
```
Comment (zuqini) [inline config.json]: are we committing this?

**Comment by zuqini** [comment]
👀

**Comment by zuqini** [comment]
LGTM I guesss

**Comment by iFallUpHill** [comment]
😤

**Pull request merged**